### DISCUSSION TITLE
Blake you are famous!

### DISCUSSION THREAD

**Discussion by mitchross** [blakeblackshear/frigate#1918] [General]
You made Linus Tech Tips! Congrats!

https://www.youtube.com/watch?v=B635wcdr6-w

**Comment by mattheys** [comment]
Yep, just watched it too, expect the issues to balloon now! 16 edge tpus though, that's a beast, wonder how much it was.

**Reply by doggyben** [reply]
$1300 on Amazon! Way overkill

**Reply by mattheys** [reply]
Not if you need to monitor 320 cameras at 5fps!
